Header header
std_msgs/ColorRGBA color
std_msgs/Float64 size
geometry_msgs/Point location
